RecoParticleFlow/PFClusterProducer/python/particleFlowBadHcalPseudoCluster_cff.py

```diff
@@ -0,0 +1,8 @@
+from RecoParticleFlow.PFClusterProducer.particleFlowBadHcalPseudoCluster_cfi import *
+
+# OFF by default, turned on via modifier for the moment
+particleFlowBadHcalPseudoCluster.enable = cms.bool(False)
+
+from Configuration.Eras.Modifier_PF_badHcalMitigation_cff import PF_badHcalMitigation
+PF_badHcalMitigation.toModify(particleFlowBadHcalPseudoCluster, enable = cms.bool(True))
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.